tools/scripts/deno/package-health.ts

```diff
@@ -0,0 +1,31 @@
+import { package_health } from 'https://deno.land/x/package_health@0.5.1/src/main.ts';
```

Comment:
what is this?

Reply:
https://github.com/guardian/actions-npm-dependencies

It's this — might be worth turning into a proper action before consumption?

Reply:
ah ha got you 👍 

going to reply on the main thread so the discussion isn't lost when this is resovled...